tests/Node.py

```diff
@@ -1039,6 +1099,46 @@ def vote(self, account, producers, waitForTransBlock=False, exitOnError=False):
 
         return self.waitForTransBlockIfNeeded(trans, waitForTransBlock, exitOnError=exitOnError)
 
+    def trxTrackWait(self, trx, waitForTransBlock=False, exitOnError=False):
+        self.trackCmdTransaction(trx)
+        return self.waitForTransBlockIfNeeded(trx, waitForTransBlock, exitOnError=exitOnError)
+
+    def stakeDelegate(self, grantor, agent, quantity):
+        action="delegate"
```

Comment:
just note: the names of some actions and parameters have changed here https://github.com/GolosChain/cyberway/pull/948

Reply:
yeah, I'll fix it separately, thx